hdwallet/src/tests.rs

```diff
@@ -130,32 +150,6 @@ mod hdwallet_tests {
 		println!("{vector_str}");
 	}
 
-	#[test]
-	fn test_derive_seed() {
```

Comment:
why did we lose this test?

Reply:
don't need it, we're already testing this in another test. it's a duplicate.